swarm/api/http/server.go

```diff
@@ -403,7 +426,11 @@ func (s *Server) HandlePostResource(w http.ResponseWriter, r *Request) {
 		Respond(w, r, err.Error(), http.StatusInternalServerError)
 		return
 	}
-	_, _, _, err = s.api.ResourceUpdate(r.Context(), r.uri.Addr, data)
+	if isRaw {
+		_, _, _, err = s.api.ResourceUpdate(r.Context(), r.uri.Addr, data)
```

Comment:
why does the function need to return a value if it is completely ignored?

Reply:
The api call might be used directly in other contexts later.